### PR TITLE
Update virtual.py

### PR DESCRIPTION
**Pull request by santosh-sudo**
**Resolved the issue of listening and recognising of the system**
https://github.com/My-Personal-HUB/AI_Jarvis/issues/2
_Have a look_

### CODE REVIEW

**Comment by shivamkumar0611** [comment]
Good work @santosh-sudo
Have good command on python and git
@ayushii18 see if any mistakes had done ..

File: virtual.py [hunk 1]
```diff
@@ -28,5 +28,21 @@ def wishMe():
...
+        audio=r.listen(source)
+
+        try:
+            statement=r.recognize_google(audio,language='en-in')
+            print(f"user said:{statement}\n")
+
+        except Exception as e:
+            speak("Pardon me, please say that again")
+            return "None"
+        return statement
```
Comment (shivambitm) [inline virtual.py:46]: _Good work @santosh-sudo_
**Have good command on python and git**
@ayushii18 see if any mistakes had done ..

**Review by ayushii18 — APPROVED** [review]
yeah nice work for today

**Review by shivambitm — APPROVED** [review]
nice

File: virtual.py (hunk 1 above)
Comment (ayushii18) [reply virtual.py:46]: No issues @shivamkumar0611 
we'll do some more coding from tommorow

**Pull request merged**